src/Blazored.Toast/Configuration/ToastSettings.cs

```diff
@@ -2,19 +2,27 @@
 {
     public class ToastSettings
     {
-        public ToastSettings(string heading, string message, string baseClass, string additionalClasses, string iconClass)
+        public ToastSettings(
+            string heading, 
+            string message,
+            IconType iconType,
+            string baseClass,
+            string additionalClasses,
+            string iconName)
         {
             Heading = heading;
             Message = message;
+            IconType = iconType;
             BaseClass = baseClass;
             AdditionalClasses = additionalClasses;
-            IconClass = iconClass;
+            IconSelector = iconName;
```

Comment:
I think could just be _icon_, I think it's clear enough and we're not dancing around if its a class for an icon or a string identifier.

```suggestion
            Icon = icon;
```

Reply:
Done